src/components/MissingSidecar/MissingSidecar.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+import { Icon } from 'patternfly-react';
+import './MissingSidecar.css';
+
+class MissingSidecar extends React.Component {
```

Comment:
Yes it's ok to use function-component isntead of class if you are not going to pass props or use state in the component. I used class because in my first iteration I was looking to style the span passing a bsStyle to the component but finally I added the style out
```js
 <span style={{ marginLeft: '10px' }}>
                <MissingSidecar />
              </span>
```

maybe in another PR I'll add this feature because I think that will be cleaner